modular_skyrat/modules/modular_implants/code/nif_persistence.dm

```diff
@@ -74,6 +81,16 @@
 	new_nif.current_theme = persistence.nif_theme
 	new_nif.is_calibrated = persistence.nif_is_calibrated
 	new_nif.rewards_points = persistence.stored_rewards_points
+
+	var/list/persistent_nifsoft_paths = list()
+	for(var/text as anything in splittext(persistence.persistent_nifsofts, "&"))
+		var/path = text2path(text)
+		if(!ispath(path))
```

Comment:
There should be a way to check that the NIFsoft is still marked as a persistent one, otherwise this could be used to still have persistent NIFsofts that would have been removed in the past, there's no sanitization on that front.

Reply:
There is, it shows via UI,